### DISCUSSION TITLE
Configuration phase 2 (Practitioner Profile configuration implementation)

### DISCUSSION THREAD

**Discussion by dubdabasoduba** [opensrp/fhircore#1342] [Ideas]
What is the vision of the configuration?
---
- We would like to achieve a configurable register flow that starts from the **Navigation drawer** to the **Client profile**.
- This will enable us to configure the following 
    - The menu item for a register and when it's displayed on the navigation drawer. 
    - The register that opens after the menu above is clicked, the configuration of the data to be displayed on the register. 
    - The profile that is opened after the register row is clicked, should also define what is displayed on the profile and the menu actions to be performed on the profile. 
- Previous steps --> [Configuration phase 2 (Client Profile configuration implementation)](https://github.com/opensrp/fhircore/discussions/1341)

User/Practitioner Profile configuration
---
**What we want to achieve**
- To create a configuration that allows us to define the user/practitioner profile. This is profile is accessed through the **Settings/Profile** menu on the Navigation Drawer

**Comment by pld** [comment]
Isn't this a version of https://github.com/opensrp/fhircore/discussions/1341?
